src/Handler/Project.hs

```diff
@@ -743,11 +744,25 @@ postUpdatePledgeR project_handle = do
 -- /t
 
 getTicketsR :: Text -> Handler Html
-getTicketsR project_handle = do
+getTicketsR project_handle = renderTicketsPage project_handle id
+
+getTicketsByTag :: Text -> Text -> Handler Html
+getTicketsByTag project_handle tag_text = renderTicketsPage project_handle (filterByTag tag_text)
```

Comment:
I appreciate that you bothered to match the existing style, but as it turns out we are trying to *change* the existing style. :) All new code should follow the [coding guidelines](https://snowdrift.coop/p/snowdrift/w/en/coding#code-style-guide), particularly regarding 80-char lines.